newdle/calendar.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+from flask import g
+from icalendar import Calendar, Event, vCalAddress, vText
+from pytz import timezone
+
+
+def create_calendar_event(newdle):
+    """Create an icalendar event based on a newdle"""
+    calendar = Calendar()
+    event = Event()
+    tz = timezone(newdle.timezone)
+    start_tz_aware = tz.localize(newdle.final_dt)
+    end_tz_aware = tz.localize(newdle.final_dt + newdle.duration)
```

Comment:
This should be enough:

```suggestion
    end_tz_aware = start_tz_aware + newdle.duration
```

Reply:
I'd just use `start_dt` and `end_dt` for these variable names

Reply:
Changed